src/views/preview/presentation.jsx

```diff
@@ -574,72 +574,91 @@ const PreviewPresentation = ({
                                             </div>
                                         ) : null}
                                     </FlexRow>
-
-                                    {/* Do not show the top-level comment form in single comment mode */}
-                                    {!singleCommentId && (
-                                        <FlexRow className="comments-root-reply">
-                                            {projectInfo.comments_allowed ? (
-                                                isLoggedIn ? (
-                                                    isShared && <ComposeComment
+                                    {isProjectCommentsGloballyEnabled ? (
```

Comment:
This part of the diff is much easier to read if you turn off whitespace diffs.

Reply:
Thanks!!